### ISSUE TITLE
MSP/IT Department Mode?

### ISSUE THREAD

**Issue by perler** [lwhitelock/HuduAutomation#5]
when we run Hudu-ADDS-Documentation.ps1 we get 

> Creating New Asset Layout
Our Companey Name was not found in Hudu

we selected "IT Department" when installing hudu - could it be, that you expect an MSP installation with multiple companies?

**Comment by lwhitelock** [comment]
Yes I haven't ever played around with Hudu in IT Department mode. I would recommend copying the $HuduBaseDomain and  $HuduAPIKey variables into powershell, then run:
New-HuduAPIKey $HuduAPIKey
New-HuduBaseUrl $HuduBaseDomain

Then run Get-HuduCompanies
I suspect that it still creates a single company in the backend, so take the name from that and set that as the $CompanyName = company at the top of the script.
